packages/tasks/src/model-libraries.ts

```diff
@@ -37,7 +37,7 @@ export enum ModelLibrary {
 	"stanza" = "Stanza",
 	"fasttext" = "fastText",
 	"stable-baselines3" = "Stable-Baselines3",
-	"ml-agents" = "ML-Agents",
+	"unity-ml-agents" = "Unity ML-Agents",
```

Comment:
```suggestion
	"ml-agents" = "Unity ML-Agents",
```

i would suggest changing only the pretty name (the one that's actually displayed) otherwise 6,414 models will be broken

Reply:
+1 this is what was suggested in the internal channel yesterday with the ML Agent's team

Reply:
Thanks, Omar just told me the same thing. I had some doubts about whether it would retrieve them or not. I update it.